src/display/api.js

```diff
@@ -933,12 +933,6 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
       intentState.renderTasks.push(internalRenderTask);
       var renderTask = internalRenderTask.task;
 
-      // Obsolete parameter support
-      if (params.continueCallback) {
```

Comment:
If this is removed, then the following JSDoc comment should also be removed: https://github.com/mozilla/pdf.js/blob/50b1a91c0273a8b419a1525827dc43e9e3f92ba0/src/display/api.js#L759-L762

Reply:
Done in the new commit.